.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+
+services:
+  - mongodb
+
+go:
+  - 1.7.1
+  - tip
+
+before_script:
+  - go get -u github.com/golang/lint/golint
+  - for package in $(go list ./... | grep -v vendor | uniq); do golint -set_exit_status $package; done
```

Comment:
I don't think we need the `| uniq` here. I haven't seen `go list` output duplicate packages.